main.go

```diff
@@ -40,23 +39,27 @@ var (
 	consumerMaxWaitTime       = flag.String("consumer-max-wait-time", "1s", "The maximum amount of time the broker will wait for Consumer.Fetch.Min bytes to become available before it returns fewer than that anyway")
 	consumerMaxProcessingTime = flag.String("consumer-max-processing-time", "1s", "The maximum amount of time the consumer expects a message takes to process")
 
-	esAddr      = flag.String("elastic-addr", "localhost:9200", "elasticsearch address (default: localhost:9200)")
+	esAddr      = flag.String("elastic-url", "http://localhost:9200", "elasticsearch url")
 	esBatchSize = flag.Int("elastic-batch-size", 1000, "maximum number of events in each bulkIndex request")
 
-	statsdAddr = flag.String("statsd-addr", "localhost:8125", "statsd address (default: localhost:8125)")
-	statsdType = flag.String("statsd-type", "standard", "statsd type: standard or datadog (default: standard)")
-	confFile   = flag.String("config", "/etc/raintank/eventtank.ini", "configuration file (default /etc/raintank/eventtank.ini")
+	statsEnabled    = flag.Bool("stats-enabled", false, "enable sending graphite messages for instrumentation")
+	statsPrefix     = flag.String("stats-prefix", "worldping.eventtank.stats.default.$hostname", "stats prefix (will add trailing dot automatically if needed)")
+	statsAddr       = flag.String("stats-addr", "localhost:2003", "graphite address")
+	statsInterval   = flag.Int("stats-interval", 10, "interval in seconds to send statistics")
+	statsBufferSize = flag.Int("stats-buffer-size", 2000, "how many messages (holding all measurements from one interval) to buffer up in case graphite endpoint is unavailable.")
+
+	confFile = flag.String("config", "/etc/raintank/eventtank.ini", "configuration file (default /etc/raintank/eventtank.ini")
 
 	logLevel   = flag.Int("log-level", 2, "log level. 0=TRACE|1=DEBUG|2=INFO|3=WARN|4=ERROR|5=CRITICAL|6=FATAL")
 	listenAddr = flag.String("listen", ":6060", "http listener address.")
 
-	eventsToEsOK   met.Count
-	eventsToEsFail met.Count
-	esPutDuration  met.Timer
-	messagesSize   met.Meter
-	msgsAge        met.Meter // in ms
-	msgsHandleOK   met.Count
-	msgsHandleFail met.Count
+	messagesSize   = stats.NewMeter32("message_size", false)
+	msgsAge        = stats.NewLatencyHistogram15s32("message_age")
```

Comment:
note that this histogram becomes inaccurate for measurements >>15s. we might want to use a Meter32 (with approx=true) instead.